### PR TITLE
Chore: Cleanup Oauth Form state variables before moving to redux

### PR DESCRIPTION
**Pull request by zeckertG**
Updating the variables that control the state for the OauthConfigForm. If there are any naming changes that look off, now would be the best time to edit those.

### CODE REVIEW

File: packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx [hunk 1]
```diff
@@ -119,54 +96,32 @@ export const OAuthConfigForm = ({
...
-    setSaved(data)
-    updateFields({
-      [BASE_URL]: base_url,
-      [WEB_URL]: looker_url,
-      [FETCH_INTENT]:
-        base_url !== '' && looker_url !== '' ? POSITIVE : CRITICAL,
-    })
-  }, [])
-
+  const [apiServerUrlValue, setApiServerUrlValue] = useState('')
```
Comment (josephaxisa) [inline packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx:99]: could these be `apiServerUrl` and `setApiServerUrl` instead?

File: packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx [hunk 2]
```diff
@@ -119,54 +96,32 @@ export const OAuthConfigForm = ({
...
+  const [fetchedUrl, setFetchedUrl] = useState('')
+  const [webUrlValue, setWebUrlValue] = useState('')
```
Comment (josephaxisa) [inline packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx:101]: ditto

File: packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx [hunk 3]
```diff
@@ -176,107 +131,111 @@ export const OAuthConfigForm = ({
...
+  const handleSaveClick = async () => {
+    const versions = await verifyUrl()
+    if (versions) {
+      saveConfig(versions.api_server_url, versions.web_server_url)
+    }
   }
 
-  const handleVerify = async (e: BaseSyntheticEvent) => {
-    await updateForm(e, false)
+  const handleVerifyClick = () => {
```
Comment (josephaxisa) [inline packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx:167]: I prefer the previous handler names, I find them more concise. `handleVerify`, `handleSave`, `handleClear`, `handleLogin` etc.

**Review by josephaxisa — COMMENTED** [review]
Left some comments. Also added @jkaster as well to review

File: packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx (hunk 1 above)
Comment (jkaster) [reply packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx:99]: I also think the `Value`suffix is redundant

File: packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx (hunk 3 above)
Comment (jkaster) [reply packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx:167]: agreed. The verb already makes the method name clear

**Review by jkaster — CHANGES_REQUESTED** [review]
I agree with dropping the `Value` and `Click` suffixes

File: packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx [hunk 4]
```diff
@@ -119,54 +96,28 @@ export const OAuthConfigForm = ({
...
+    setMessageBarText(message)
   }
 
   const isConfigured = () => {
-    return (
-      saved !== EmptyConfig &&
-      fields[BASE_URL] === saved.base_url &&
-      fields[WEB_URL] === saved.looker_url
-    )
+    return apiServerUrl === saved.base_url && webUrl === saved.looker_url
```
Comment (zeckertG) [inline packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx:115]: comparing the EmptyConfig always results in false, as objects are passed / compared by reference in JS.

Comment (jkaster) [reply packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx:115]: I'm guessing empty urls can't be saved?

**Review by jkaster — CHANGES_REQUESTED** [review]
Please drop the `Click` suffix

**Review by jkaster — APPROVED** [review]
LGTM

We may want to change the names of the stored config properties in the future, but not required right now.

**Review by josephaxisa — APPROVED** [review]
Looks good!

**Pull request merged**